packages/dinero.js/etc/dinero.js.api.md

```diff
@@ -0,0 +1,173 @@
+## API Report File for "dinero.js"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).

```

Comment:
This file is generated by api-extractor. From what I understand it is generally used as a way to quickly highlight API changes for project maintainers. Making it easy to see if a contribution modifies the external api.